tests/codemods/test_flask_json_response_type.py

```diff
@@ -0,0 +1,238 @@
+from core_codemods.flask_json_response_type import FlaskJsonResponseType
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from textwrap import dedent
+
+
+class TestFlaskJsonResponseType(BaseCodemodTest):
+    codemod = FlaskJsonResponseType
+
+    def test_name(self):
+        assert self.codemod.name() == "flask-json-response-type"
+
+    def test_simple(self, tmpdir):
+        input_code = """\
+        from flask import make_response, Flask
+        import json
+
+        app = Flask(__name__)
+
+        @app.route("/test")
+        def foo(request):
+            json_response = json.dumps({ "user_input": request.GET.get("input") })
+            return make_response(json_response)
+        """
+        expected = """\
+        from flask import make_response, Flask
+        import json
+
+        app = Flask(__name__)
+
+        @app.route("/test")
+        def foo(request):
+            json_response = json.dumps({ "user_input": request.GET.get("input") })
+            return (make_response(json_response), {'Content-Type': 'application/json'})
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_simple_return_json(self, tmpdir):
+        input_code = """\
+        from flask import Flask
+        import json
+
+        app = Flask(__name__)
+
+        @app.route("/test")
+        def foo(request):
+            json_response = json.dumps({ "user_input": request.GET.get("input") })
+            return json_response
```

Comment:
I think this does not need to be modified as flask will automatically set the correct content type in this case.

Reply:
It won't. The return of `json_response` is a string and `flask` will only apply `jsonify` for dict returns.